tests/unit/test_charm.py

```diff
@@ -70,7 +70,7 @@ def test_main_no_relation(
         "namespace": None,
         "port": 5556,
         "config_yaml": json.dumps(config_yaml),
-        "config_hash": "1419b40a8f223615121b8b1ffdb83a689c5da15e16b9b7fd71c192a544295995",
+        "config_hash": "de746b7609029af76f80a7daa43df83121715d1434baf13a36b32b02220b01b9",
```

Comment:
Why do these need to be updated?

Reply:
The hash is updated with the configs. Adding `http://` to the issuer changed the hash